app/forms.py

```diff
@@ -0,0 +1,29 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, FloatField, FileField, TextField, MultipleFileField, TextAreaField
+from wtforms.validators import ValidationError
+from wtforms.validators import DataRequired, Length
+
+
+class FormNewServices(FlaskForm):
+    name = TextField('name')
+    description = TextAreaField('description')
+    price = FloatField('price')
```

Comment:
the `price` should be a dict object, not one value, yesterday I shared this doc that illustrates more info about services.. you can double check it from here https://docs.google.com/document/d/1se7TXX3YcZYdZ8TX9MG3ILQFgQIxwv6GWzLdW5Zs1tE/edit?usp=sharing